src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,20 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+
 public class UserService {
 
     public int getUserScore(String[] records, String email) {
-        return 0;
+
+        for (String record : records) {
+            if (record.substring(0,record.indexOf(":")).equals(email)) {
+                return Integer.parseInt(record.substring(record.indexOf(":") + 1,
```

Comment:
what if userScore is 3232 and you get substring 1  3 ? 
user method split() instead and save result to String[] yourVariable